### PR TITLE
Fixing Signal Sciences - test playbook + code changes

### PR DESCRIPTION
**Pull request by guykeller**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24934

## Description
1. Fixing Signal Science's TPB
2. Adding certificate and proxy handling and support
3. Moving out of skip

### CODE REVIEW

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 commonfields:
-  id: Signal Sciences WAF
+  id: Signal Sciences WAF_copy
   version: -1
-name: Signal Sciences WAF
-display: Signal Sciences WAF
+name: Signal Sciences WAF_copy
+display: Signal Sciences WAF_copy
```
[the remark this answers is not in the document]
Comment (guykeller) [reply Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.yml]: I think you commented before I already changed it in the last commit 👍

File: Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.yml [hunk 2]
```diff
@@ -21,6 +21,16 @@ configuration:
...
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
```
Comment (bakatzir) [inline Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.yml:33]: please put this at the bottom of the configuration

File: Packs/SignalSciences/ReleaseNotes/1_0_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Signal Sciences WAF
+- Added support for proxy usage and secure requests.
```
Comment (bakatzir) [inline Packs/SignalSciences/ReleaseNotes/1_0_2.md:4]:
```suggestion
Added the **proxy**, **Trust any certificate** integration parameters.
```

File: Packs/SignalSciences/TestPlaybooks/playbook-SignalSciencesTest.yml [hunk 4]
```diff
@@ -1,14 +1,15 @@
 id: SignalSciences-Test
-version: -1
+version: 11
+vcShouldKeepItemLegacyProdMachine: false
```
Comment (bakatzir) [inline Packs/SignalSciences/TestPlaybooks/playbook-SignalSciencesTest.yml:3]:
```suggestion
version: -1
fromversion: 5.0.0
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**